js/index.js

```diff
@@ -42,7 +50,71 @@ let currentPosition = 0;
 const carrouselArrows = document.querySelectorAll(".carrousel__arrow");
 /* Take actual image from document */
 const carrouselImage = document.querySelector(".carrousel__img");
+
+ function automatic() {
+  if (currentPosition < image.length - 1) {
+    carrouselImage.src = image[currentPosition + 1];
+    currentPosition++;
+  } else {
+    carrouselImage.src = image[0];
+    currentPosition = 0;
+  }
+  ovalBlack
+  ovalBlack();
+};
+/*when the active page loads the interval function and sends the data and the activation
+ to the automatic function*/
```

Comment:
Remember the structure of comments. 
I think it's this way:
/**
*example
*/
If  I'm wrong please correct me.

Reply:
@hanna446 is right, please update the comment structure.